src/wwwroot/partials/settings/domain-manager.html

```diff
@@ -8,18 +8,18 @@ <h2 class="width--full">{{'domain-manager_title' | translate}}</h2>
       <button class="button button--large domain--input-toggler" ng-click="vm.showNewDomainInput()">{{'domain-manager_button' | translate}}</button>
     </div>
   </div>
-  <form method="post" id="form" name="form" ng-submit="vm.addDomain(form)" novalidate>
+  <form method="post" id="form" name="form" novalidate>
```

Comment:
I liked the call to controller in `form` in place of `button`, why did you do this change?